docs: adds docs.slack.dev page (#100)
slackapi/slack-mcp-plugin: docs/slack-skills-plugin.md

# Slack MCP and Skills Plugin

> The Slack MCP and Skills Plugin for AI tools bundles together a set of skills that help you develop on the Slack platform with the [Slack MCP Server](/ai/slack-mcp-server). You can use the plugin with Claude Code and Cursor.

Reviewer: The goal is to add support for more AI tools in the near future 🙏 

```suggestion
The Slack MCP and Skills Plugin for AI tools bundles together a set of skills that help you develop on the Slack platform with the [Slack MCP Server](/ai/slack-mcp-server).
```

Author: I'll just update the docs when those come in !

Installing the plugin sets up two things:

* **[Skills](#skills)**. Skills supercharge you and your agents when developing Slack apps.
* **[Slack MCP Server connection](/ai/slack-mcp-server)**. The Slack MCP server lets you and your agent interact directly with your Slack workspace, such as searching channels, sending messages, and managing canvases.

> The Slack MCP server is configured automatically when the plugin loads. You'll be prompted to authenticate into your Slack workspace via OAuth. For full setup details, see [Connect to Claude](/ai/slack-mcp-server/connect-to-claude).

Reviewer: ```suggestion
The Slack MCP server is configured automatically when the plugin loads. You'll be prompted to authenticate into your Slack workspace via OAuth. Full setup details vary depending on the AI tool you are using.
```

---

## Installing the plugin

> You can install the plugin for Claude Code or for Cursor.

Reviewer: What do you think about removing this line? 🤔

Author: Are we trying to avoid specifically calling out which tools? My initial thought is people will come here thinking "i use X tool, does the plugin work for that"

The plugin is published on the [official Cursor Marketplace](https://cursor.com/marketplace/slack). You can install the plugin by clicking the nifty button below:

[![Install MCP Server](https://cursor.com/deeplink/mcp-install-dark.svg)](https://cursor.com/en-US/install-mcp?name=slack&config=eyJ1cmwiOiJodHRwczovL21jcC5zbGFjay5jb20vbWNwIiwiYXV0aCI6eyJDTElFTlRfSUQiOiIzNjYwNzUzMTkyNjI2Ljg5MDM0NjkyMjg5ODIifX0=")

---

## Using skills {/* #skills */}

Skills are focused sets of instructions and references that your assistant loads when a task calls. The skills load automatically when your prompt calls for them.

Most of the skills work on their own, without a connection to the [Slack MCP server](/ai/slack-mcp-server). The one exception is the `slack-search` skill, which relies on the Slack MCP server to query your workspace.

| Skill | What it helps with | Example prompt |
|-------|--------------------|----------------|
| `block-kit` | Build and validate [Block Kit](/block-kit) layouts for messages, modals, and Home tabs, validating against the `blocks.validate` API method. | _"Build a Block Kit modal with a name field, a dropdown to pick a channel, and a submit button."_ |

> | `create-slack-app` | Scaffold a new Slack app or agent with the [Slack CLI](/tools/slack-cli) and [Bolt](/tools/bolt) (JavaScript or Python). | _"Scaffold a new Bolt for JavaScript app that listens for the `app_mention` event."_ |

Reviewer 2: the bolt link has to be bolt-js or bolt-python

> | `create-slack-app` | Scaffold a new Slack app or agent with the [Slack CLI](/tools/slack-cli) and [Bolt](/tools/bolt) (JavaScript or Python). | _"Scaffold a new Bolt for JavaScript app that listens for the `app_mention` event."_ |

Author: ```suggestion
| `create-slack-app` | Scaffold a new Slack app or agent with the [Slack CLI](/tools/slack-cli) and [Bolt](/tools#bolt) (JavaScript or Python). | _"Scaffold a new Bolt for JavaScript app that listens for the `app_mention` event."_ |
```
